resources/language/resource.language.en_gb/strings.po

```diff
@@ -933,3 +933,15 @@ msgstr ""
 msgctxt "#33197"
 msgid "A sync is already running, please wait until it completes and try again."
 msgstr ""
+
+msgctxt "#33198"
+msgid "Select the libraries to update"
+msgstr ""
+
+msgctxt "#33199"
+msgid "Select the libraries to repair"
```

Comment:
Is it required to add translations in the wrong language to all files, or will the client fallback to English when a translation doesn't exist?

Reply:
I'm sure it would use english if the translation doesn't exists. I haven't used it before but I believe weblate will see them in the other language files and makes them available for translation in the interface

Reply:
If it will fallback to English I would say to just not include the additions to the other language files and let Weblate handle that automatically, but either way is fine.

Reply:
I wasn't sure if we late would be able to determine from the English string file that the new ones should be in the others. Also what python would do if a user is using a language that hasn't been translated thus no listing for it would be found in the string file, I think it only defaults to English because we're using the Msgid wrong

Reply:
On Android it's common practice to leave untranslated strings out of the file, but I will defer to you since I know nothing about how Kodi handles translations.

Reply:
After doing some testing, it seems Kodi just displays the English text if no translation is found in the other files. I have thus removed them